db/schema.rb

```diff
@@ -11,17 +11,28 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160503184017) do
+ActiveRecord::Schema.define(version: 20160503231211) do
+
+  create_table "features", force: :cascade do |t|
+    t.string   "name"
+    t.integer  "user_id",    null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  add_index "features", ["user_id"], name: "index_features_on_user_id"
 
   create_table "metrics", force: :cascade do |t|
     t.string   "name"
     t.string   "definition"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.integer  "user_id"
+    t.integer  "user_id",    null: false
```

Comment:
I'm not supposed to edit schema.rb directly, only update the schema using migrations? That makes sense, I wasn't sure what to do once I started adding in the Features model so I just tried to update the schema directly. Thank you!